src/icp.cpp

```diff
@@ -121,7 +121,7 @@ usage() {
         "icp copies an irods data-object (file) or collection (directory) to another",
         "data-object or collection.",
         " ",
-        "The -Q option specifies the use of the RBUDP transfer mechanism which uses",
+        "The -Q option [Deprecated] specifies the use of the RBUDP transfer mechanism which uses",
```

Comment:
Maybe this could be a separate sentence, for emphasis? Like...

```suggestion
        "Note: The -Q option has been deprecated.",
        "The -Q option specifies the use of the RBUDP transfer mechanism which uses",
```

Reply:
I made these changes.